tg_utils/health_check/checks/phantomjs/backends.py

```
@@ -0,0 +1,39 @@
+from django.conf import settings
+
+from tg_utils.health_check.base_checks import HTTPBasedHealthCheck
+
+
+class PhantomJSHealthCheck(HTTPBasedHealthCheck):
+    """Check if phantomJS pdf generator does not fail with error"""
+    expected_status_code = 200
+    method = 'POST'
+
+    def get_payload(self):
+        html = """
+            <html>
+                <body>
+                    <h1>Health is important</h1>
+                    <div>
+                        If you are a developer reading this, take a 5 minute break to walk around now,
+                        and prolong your life.
+                    </div>
+                </body>
+            </html>
+        """
+
+        return {
+            'html': html,
+        }
+
+    def get_url(self):
+        return settings.PHANTOMJS_URL
```

would be better to raise error with descriptions if this setting is missing